components/LanguageSwitcher/src/LanguageSwitcherTitle.tsx

```diff
@@ -0,0 +1,16 @@
+import React, { HTMLAttributes } from 'react';
+import clsx from 'clsx';
+
+export type LanguageSwitcherTitleProps = HTMLAttributes<HTMLDivElement>;
+
+export const LanguageSwitcherTitle: React.FC<LanguageSwitcherTitleProps> = (props: LanguageSwitcherTitleProps) => {
+  const className = clsx('denhaag-language-switcher__title', props.className);
+
+  return (
+    <h4 {...props} className={className}>
```

Comment:
Hmm I don't know if this should always be a `h4`... okay for now but we might want to revisit later :)